pkg/client/injection/informers/factory/factory.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Knative Authors
+Copyright 2020 The Knative Authors
```

Comment:
super nit, my understanding was that for existing files, we should not be going around updating the copyright years...

Reply:
this is all generated code and I updated the boilerplate. 🤷‍♂